app/controllers/application_controller.rb

```diff
@@ -21,6 +21,8 @@ def homepage_path_for_role(educator)
       educators_districtwide_url
     elsif educator.schoolwide_access? || educator.has_access_to_grade_levels?
       school_url(educator.school)
+    elsif educator.school.school_type == 'HS'
+      default_section_path(educator)
```

Comment:
If an educator is tied to a high school, they will be defaulted to the section view. We may want to consider defaulting to section view for elementary school educators who have no homerooms (this would allow art, music, etc teachers to see their students).